tests/components/vicare/snapshots/test_number.ambr

```diff
@@ -565,3 +565,60 @@
     'state': 'unavailable',
   })
 # ---
+# name: test_all_entities[number.model0_temperature-entry]
```

Comment:
Is this name as well as `entity_id` and `original_name` related to the device class? I would have expect this to be `DHW temperature`?

Reply:
Looks like this is not generated correctly for me locally. I could fix this manually but would like to know why this is happening.

Reply:
@joostlek any idea why this failed locally? Missing translation?

Reply:
Did you generate translations?

Reply:
No, just the `string.json`, the `entity_id` contains the translated name?

Reply:
`python3 -m script.translations develop --all`

HA takes the strings from `translations/en.json`